holoviews/streams.py

```diff
@@ -223,3 +223,60 @@ class PositionXY(Stream):
     def __init__(self, preprocessors=[], source=None, subscribers=[], **params):
         super(PositionXY, self).__init__(preprocessors=preprocessors, source=source,
                                         subscribers=subscribers, **params)
+
+
+
+
+class ParamValues(Stream):
+    """
+    A Stream based on the parameter values of some other parameterized
+    object, whether it is a parameterized class or a parameterized
+    instance.
+
+    The update method enables the stream to update the parameters of the
+    specified object.
+    """
```

Comment:
Right. Though if you aren't using update to actually update parameters, you can use ``Stream.trigger([callback])`` instead. For paramnb it would be more awkward than update as you would need to wrap that in a lambda or something.